src/controllers/HomeController.ts

```diff
@@ -1,5 +1,5 @@
 import express from "express";
 
 export const getIndexPage = (req: express.Request, res: express.Response) => {
-    res.render('index.njk', { token: req.session.token});
+    res.render('index.njk', { token: req.session.token, role: res.locals.role as string});
```

Comment:
Role should already be accessible

Reply:
This has been corrected, checked, and committed.